src/napari_imagej/widgets/menu.py

```diff
@@ -41,7 +43,12 @@ def __init__(self, viewer: Viewer):
         self.gui_button: GUIButton = GUIButton(viewer)
         self.layout().addWidget(self.gui_button)
 
+        self.repl_button: REPLButton = REPLButton(viewer)
+        self.repl_button.setToolTip("Show/hide the SciJava REPL")
+        self.layout().addWidget(self.repl_button)
+
         self.settings_button: SettingsButton = SettingsButton(viewer)
+        self.settings_button.setToolTip("Show napari-imagej settings")
```

Comment:
Generally I've been putting the tool tips in the constructors of each button - could we move these there for consistency?

Thanks so much for adding the tooltip to the settings button - can't believe I forgot that!

Reply:
Moved in [6dcacbf](https://github.com/imagej/napari-imagej/pull/265/commits/6dcacbfbe49f0ba256259c1dea77bfe79c7dcf13)